BasicAndroidAccessibility/app/src/main/res/layout/activity_main.xml

```diff
@@ -13,51 +13,29 @@
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
--->
-
+ -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:paddingBottom="@dimen/activity_vertical_margin"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
-    tools:context="com.example.android.basicandroidaccessibility.MainActivity">
-
-    <Button
-        android:id="@+id/activity_content_labeling_button"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:onClick="launchActivity"
-        android:text="@string/title_activity_content_labeling" />
+    android:fitsSystemWindows="true"
+    tools:context=".MainActivity">
 
-    <Button
-        android:id="@+id/activity_content_grouping_button"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:onClick="launchActivity"
-        android:text="@string/title_activity_content_grouping" />
+        <androidx.appcompat.widget.Toolbar
```

Comment:
In your activity to set up the toolbar, use Navigation UI's methods to set it up:
https://developer.android.com/reference/kotlin/androidx/navigation/ui/NavigationUI#setupActionBarWithNavController(androidx.appcompat.app.AppCompatActivity,%20androidx.navigation.NavController)